content/en/docs/v2/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: v2 docs
+cascade:
+  version: v2
+  versName: &name v2
+  git_version_tag: v2
+linkTitle: *name
+simple_list: true
+weight: -350 # Weight for doc version vX.Y should be -XY0
```

Comment:
Weight should be -200 if we're following the same convention.

Reply:
Good catch, thx